src/imu/mpu9255/mpu9255.hpp

```diff
@@ -0,0 +1,55 @@
+#ifndef SRC_MPU9255_HPP_
+#define SRC_MPU9255_HPP_
+
+#include "accelerometer.hpp"
```

Comment:
why do you need both, the concrete implementation and the virtual interface?

Reply:
Thats really ugly. But I need the concrete implementation, because the class Mpu9255 needs to set up the single sensors. I cannot inject them from the outside. I would need to create a Factory for each single sensor, and I think that would be overengineering.
But I also need the virtual interfaces to inject the mock objects for the unittests. Thats the reason, why I need to include both here. Concrete for production code, virtual interface for unittest seams.